genai_bench/user/openai_user.py

```diff
@@ -261,6 +261,29 @@ def parse_chat_response(
                 num_prefill_tokens, num_prompt_tokens, tokens_received = (
                     self._get_usage_info(data, num_prefill_tokens)
                 )
+                if not time_at_first_token:
+                    choices = data["choices"]
+                    if len(choices) > 0:
```

Comment:
A bit confused by the logic here, we have `if not data["choices"]` at L256. Why would here be true?